resource_management_mobile/src/app/auth/login/login.page.html

```diff
@@ -0,0 +1,53 @@
+<!-- <ion-header>
+  <ion-toolbar>
+    <ion-title>login</ion-title>
+  </ion-toolbar>
+</ion-header> -->
+
+<ion-content [fullscreen]="true" >
+  <!-- <ion-header collapse="condense">
+    <ion-toolbar>
+      <ion-title size="large">login</ion-title>
+    </ion-toolbar>
+  </ion-header> -->
+  <section style="height: 100%;">
+    <div class="login_background">
```

Comment:
Avoid using div, use ion tags to fulfill the design

Reply:
Instead of div, I used ion-grid tags.